lib/utils/matrix_sdk_extensions/flutter_hive_collections_database.dart

```diff
@@ -1,20 +1,25 @@
 import 'dart:convert';
 import 'dart:io';
 
-import 'package:fluffychat/utils/platform_infos.dart';
-import 'package:fluffychat/utils/storage_directory_utils.dart';
 import 'package:flutter/foundation.dart' hide Key;
 import 'package:flutter/services.dart';
 import 'package:flutter_secure_storage/flutter_secure_storage.dart';
 import 'package:hive/hive.dart';
 import 'package:matrix/matrix.dart';
 import 'package:path_provider/path_provider.dart';
 
+import 'package:fluffychat/domain/model/keychain_sharing_restore_token.dart';
+import 'package:fluffychat/domain/model/keychain_sharing_session.dart';
+import 'package:fluffychat/utils/platform_infos.dart';
+import 'package:fluffychat/utils/storage_directory_utils.dart';
+
 class FlutterHiveCollectionsDatabase extends HiveCollectionsDatabase {
+  final String? userId;
```

Comment:
IMO it is a good solution, it can make our app can not support multiple account

Reply:
removed, should update later when multi account supported